src/kOS.Safe/Exceptions/KOSOnceInvalidHereException.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace kOS.Safe.Exceptions
+{
+    /// <summary>
+    /// A version of KOSCommandInvalidHere describing an attempt to use
+    /// the WAIT keyword when in a trigger.
+    /// </summary>
+    public class KOSOnceInvalidHereException : KOSCommandInvalidHereException
+    {
+        public override string HelpURL
+        {
+            get { return "http://ksp-kos.github.io/KOS_DOC/summary_topics/CPU_hardware/index.html#WAIT"; }
```

Comment:
crap.  this is a cut and paste error I didn't catch.  It's going to direct users to the wrong part of the docs because this is a copy from a different exception.